client/state/selectors/is-requesting-billing-transaction.js

```diff
@@ -0,0 +1,31 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingBillingTransactions } from 'state/selectors';
+
+/**
+ * Returns true if we are currently making a request to get the billing transactions.
```

Comment:
This needs to be updated to correspond to the fact that this selector is responsible for requesting single transactions, and not for requesting all.